courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2501 @@
+{
```

Comment:
<p>Got this error, any ideas?</p><p> </p><pre class="ql-syntax" spellcheck="false">slice index 0 of dimension 0 out of bounds. for 'strided_slice' (op: 'StridedSlice') with input shapes: [0], [1], [1], [1] and with computed input tensors: input[1] = &lt;0&gt;, input[2] = &lt;1&gt;, input[3] = &lt;1&gt;.
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>I think this error was due to scalar function not working as a result of indenting the code based on the Pep8 code check I ran.  Once I put the scalar function in its original state, this error message did not appear.</p>

Reply:
<p>Hi Danielle - First, regarding the datatype float warning, I implemented the recommended solution to remove the warning but it did not go away.  From my research, this is a bug in the Keras sdk.  See this link:  <a href="https://github.com/tensorflow/tensorflow/issues/34849" target="_blank">https://github.com/tensorflow/tensorflow/issues/34849</a></p><p> </p><p>In terms of the "slice index out of bounds error".  Once I corrected the scale function, this error disappeared.  </p><p> </p><p>Summary:  The PEP8 code check broke a few of the functions and once brought to their original state, worked as before.  </p><p> </p><p>I have uploaded the latest version of the notebook.</p><p> </p><blockquote>
</blockquote><p> </p><p> </p>